tests/test_real_gemini_integration.py

```
@@ -0,0 +1,268 @@
+"""
+実際のGemini APIを使用した統合テスト
+CLAUDE.mdの原則に従い、モックは一切使用しない
+"""
+import pytest
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from app import app, initialize_llm
```

Ruff (0.12.2)

`app.initialize_llm` imported but unused

Remove unused import: `app.initialize_llm`

(F401)